src/test/java/io/confluent/connect/hdfs/avro/HiveIntegrationAvroTest.java

```diff
@@ -239,17 +230,20 @@ public void testHiveIntegrationFieldPartitionerAvro() throws Exception {
     Table table = hiveMetaStore.getTable(hiveDatabase, TOPIC);
 
     List<String> expectedColumnNames = new ArrayList<>();
-    for (Field field: schema.fields()) {
+    for (Field field : schema.fields()) {
       expectedColumnNames.add(field.name());
     }
+    Collections.sort(expectedColumnNames);
 
     List<String> actualColumnNames = new ArrayList<>();
-    for (FieldSchema column: table.getSd().getCols()) {
+    for (FieldSchema column : table.getAllCols()) {
```

Comment:
let's add a comment here, since you discovered that this is required to surface duplicate columns.